composer.json

```diff
@@ -15,14 +15,13 @@
     ],
     "require": {
         "php": ">=7.3",
-        "illuminate/database": "^5.5|^6|^7",
-        "illuminate/http": "^5.5|^6|^7",
-        "illuminate/routing": "^5.5|^6|^7",
-        "illuminate/support": "^5.5|^6|^7",
+        "illuminate/database": ">=5.5",
```

Comment:
Could you do this change in a different PR? This has nothing to do with removing Guzzle. Also, please keep it like `^5.5|^6|^7|^8`, [we are discussing](https://github.com/arkaitzgarro/elastic-apm-laravel/issues/94) which version to support in the new v2.x release, we will address the right composer required format in the consequent PR.

Reply:
ok